app/api/routers/breeze_buddy/pod.py

```
@@ -0,0 +1,206 @@
+    (gold, standard, overflow, shared, merchant pools).
+    """
+    if not ENABLE_VOICE_AGENT_POD_ISOLATION:
+        return {"error": "Pod isolation not enabled"}
+
+    try:
+        router_client = get_smart_router_client()
+        pool_status = await router_client.get_pool_status()
+        return {
+            "status": "ok",
+            "pools": pool_status,
+        }
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)